src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -82,16 +87,38 @@ private static SymbolLookup makeSystemLookup() {
         }
     }
 
+    /* Obtain the address of the inlined function list by loading the specified library. */
+    private static MemorySegment getInlinedFunctListAddr() {
+        /* Directly load the specified library only once with the inlined libc functions
+         * missing in the default library.
+         */
+        SymbolLookup funcsLibLookup = libLookup(libs -> libs.load(jdkLibraryPath("syslookup")));
+
+        /* Perform a FFI downcall to obtain the address of the inlined function list. */
+        MemorySegment funcs_addr = funcsLibLookup.findOrThrow("funcs_addr");
```

Comment:
`findOrThrow()` is only available in the head stream. Please revert to:
```java
        MemorySegment funcs_addr = funcsLibLookup.find("funcs_addr").orElseThrow();
```

Reply:
Just notice and reverted.